README.md

```diff
@@ -94,7 +94,6 @@ We're definitely not doing that. With react_on_rails, webpack is mainly generati
   
   ```ruby
   gem "react_on_rails"
-  gem "therubyracer"
   ```
```

Comment:
I realize that we add it with the generator, but it's probably important enough to mention it.